modcc/solvers.cpp

```diff
@@ -254,27 +264,26 @@ void SparseSolverVisitor::visit(AssignmentExpression *e) {
         // Otherwise, for non-zero coefficient c, the entry is -c*dt.
 
         if (r.coef.count(dvars_[j])) {
-            expr = make_expression<MulBinaryExpression>(loc,
-                       r.coef[dvars_[j]]->clone(),
-                       dt_expr->clone());
+            expr = steadystate_ ? r.coef[dvars_[j]]->clone() :
+                    make_expression<MulBinaryExpression>(loc, r.coef[dvars_[j]]->clone(), dt_expr->clone());
 
```

Comment:
Could you expand the comment above (lines 262–264) to describe the steady-state case?

Reply:
Yep. Thanks for pointing it out.